.gitignore

```diff
@@ -102,3 +102,7 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# ide
```

Comment:
is it required?

Reply:
Yep. This is certainly useful for people working on VSCode.